### ISSUE TITLE
salt-ssh: --user option missing from the cli documentation

### ISSUE THREAD

**Issue by rossengeorgiev** [saltstack/salt#44313]
### Description of Issue/Question
`--user` not on https://docs.saltstack.com/en/latest/ref/cli/salt-ssh.html

### Steps to Reproduce Issue
```bash
root@master:/vagrant# salt-ssh --help | grep user
    --user=SSH_USER     Set the default user to attempt to use when
```

### Versions Report
```yaml
Salt Version:
           Salt: 2017.7.1

Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Nov 19 2016, 06:48:10)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: Ubuntu 16.04 xenial
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-92-generic
         system: Linux
        version: Ubuntu 16.04 xenial
```

**Comment by Ch3LL** [comment]
seems easy enough to add mind taking a stab at a PR for it? looks like its in this file: doc/ref/cli/salt-ssh.rst

**Comment by rallytime** [comment]
Closed via #44322
